src/Components/MovieShowPage/MovieShowPage.js

```diff
@@ -20,6 +20,18 @@ export class MovieShowPage extends Component {
 
   submitRating = (rating, movieId, userId) => {
     postRating(rating, movieId, userId)
+      .then(data => {
+        getUserRatings(userId)
+          .then(ratings => {
+            this.props.getRatings(ratings.ratings);
+            fetchMovies()
+              .then(data => {
+                this.props.getMovies(data.movies)
+             })
+             .catch(err => console.log(err))
+          })
+          .catch(err => console.log(err))
+      })
```

Comment:
Would have loved to test this if we had more time, but I'm still really happy with all the testing practice we did on this project!

Reply:
I agree